setup.py

```diff
@@ -83,18 +83,20 @@
     "filelock",
     "flake8>=3.8.3",
     "hf-doc-builder>=0.3.0",
-    "huggingface-hub>=0.8.1,<1.0",
+    "huggingface-hub>=0.8.1",
```

Comment:
Think we don't have to be too restrictive with the Hub here - we'll test before 1.0 comes out